.github/CODEOWNERS

```diff
@@ -4,9 +4,45 @@
 #
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
+# BRE for publish workflow changes
+.github/workflows/publish-*.yml @bitwarden/dept-bre
+
+# Shared workflows ownership
+
+## BRE & SM teams shared ownership
+.github/workflows/build-cli-docker.yml @bitwarden/dept-bre @bitwarden/team-secrets-manager-dev
+.github/workflows/build-cli.yml @bitwarden/dept-bre @bitwarden/team-secrets-manager-dev
+.github/workflows/build-cpp.yml @bitwarden/dept-bre @bitwarden/team-secrets-manager-dev
+.github/workflows/build-dotnet.yml @bitwarden/dept-bre @bitwarden/team-secrets-manager-dev
+.github/workflows/build-go.yml @bitwarden/dept-bre @bitwarden/team-secrets-manager-dev
+.github/workflows/build-java.yml @bitwarden/dept-bre @bitwarden/team-secrets-manager-dev
+.github/workflows/build-napi.yml @bitwarden/dept-bre @bitwarden/team-secrets-manager-dev
+.github/workflows/build-python-wheels.yml @bitwarden/dept-bre @bitwarden/team-secrets-manager-dev
+.github/workflows/build-ruby.yml @bitwarden/dept-bre @bitwarden/team-secrets-manager-dev
+.github/workflows/build-rust-crates.yml @bitwarden/dept-bre @bitwarden/team-secrets-manager-dev
```

Comment:
This is listed in the `## Multiple owners` section below as well.  Could you delete one of the entries?

Reply:
Good catch. Removed it from here, as this workflow is not only owned by SM dev.